### DISCUSSION TITLE
Detectron2 bounding box format

### DISCUSSION THREAD

**Discussion by unrue** [facebookresearch/detectron2#5066] [General]
Hi, 
about object detection, which bounding box format detectron2 use? [X,Y,W,H] or [X,Y,XMAX,YMAX] ? Thanks

**Comment by ravijo** [comment]
As far as I remember, I think following two formats are supported:
`BoxMode.XYXY_ABS`, and `BoxMode.XYWH_ABS`

Source: https://detectron2.readthedocs.io/en/latest/tutorials/datasets.html

Below is an example:
```
obj = {
    "bbox": [np.min(px), np.min(py), np.max(px), np.max(py)],
    "bbox_mode": BoxMode.XYXY_ABS,
    "segmentation": [poly],
    "category_id": 0,
    "iscrowd": 0
}
```

Source: https://gilberttanner.com/blog/detectron-2-object-detection-with-pytorch/

**Comment by unrue** [comment]
Ok, but what is the default mode? My training get AP=25 when I use X,Y,W,H  and only 8 when the format is X,Y,XMAX,YMAX

So I suppose the default is the first one.
